dash/development/_r_components_generation.py

```diff
@@ -204,18 +204,16 @@ def generate_class_string(name, props, project_shortname, prefix):
         for p in prop_keys
     )
 
-    return r_component_string.format(
-        prefix=prefix,
-        name=name,
-        default_argtext=default_argtext,
-        wildcards=wildcards,
-        wildcard_declaration=wildcard_declaration,
-        default_paramtext=default_paramtext,
-        project_shortname=project_shortname,
-        prop_names=prop_names,
-        wildcard_names=wildcard_names,
-        package_name=package_name,
-    )
+    return r_component_string.format(funcname=format_fn_name(prefix, name),
```

Comment:
this indent alignment is not pythonic, is this by `black`?

Reply:
@byronz Looks plenty Pythonic to me, that's four spaces 🙂 

The indentation is not from `black`; it dates back to the initial commit of this method from six months ago.

Reply:
Are you talking about the lines below? That's one of the standard patterns, right? `name` and everything below it aligns with `funcname`.

Sounds like we should just use black and stop discussing this stuff. But let's wait until after 1.0 to do this.